files/startup_script_jenkins.sh

```diff
@@ -49,4 +49,11 @@ sed -i "s/subnet-id/$SUBNET_ID/g" /data/jenkins/jenkins_home/config.xml
 ## Run docker
 cd /data/docker
 sudo docker build -t cicd-jenkins -f jenkins.dockerfile .
-sudo docker-compose up -d
\ No newline at end of file
+sudo docker-compose up -d
```

Comment:
We dont need to start docker compose, and start service in one time. This line should be removed